### PR TITLE
quick block quote test

### PR DESCRIPTION
**Pull request by catlook**
A starter for more use cases, but quick syntax check

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-c63141691e/whitepapers/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/support.rackspace.com/build-5dbcd3daf7/whitepapers/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-6923dc52f3/whitepapers/)

**Comment by catlook** [comment]
@nimbinatus , let's go ahead and merge this.  I'm shifting gears temporarily, so remaining use cases will come a bit later.  Let me know what to fix, and let's get 'er done!

File: content/use-cases/oakley-re-platforms-infrastructure-online-to-omnichannel.md [hunk 1]
```diff
@@ -0,0 +1,162 @@
+---
+permalink: aoakley-re-platforms-infrastructure-online-to-omnichannel/
```
Comment (nimbinatus) [inline content/use-cases/oakley-re-platforms-infrastructure-online-to-omnichannel.md]: random `a` at the beginning of the link?

File: content/use-cases/omni-channel-commerce-in-the-cloud.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
+---
+permalink: a-omni-channel-commerce-in-the-cloud/
```
Comment (nimbinatus) [inline content/use-cases/omni-channel-commerce-in-the-cloud.md]: random `a-` at the beginning of the link?

**Comment by nimbinatus** [comment]
@catlook, also, a message from Strider: 
>Couldn't find file 'use-cases/mapmyfitness-fortifies-growth-saves-cost-without-sacrificing-performance/MapMyFitnesslogo.jp' in content/use-cases/mapmyfitness-fortifies-growth-saves-cost-without-sacrificing-performance.md

File: content/use-cases/omni-channel-commerce-in-the-cloud.md (hunk 2 above)
Comment (catlook) [inline content/use-cases/omni-channel-commerce-in-the-cloud.md]: fixed

File: content/use-cases/oakley-re-platforms-infrastructure-online-to-omnichannel.md (hunk 1 above)
Comment (catlook) [inline content/use-cases/oakley-re-platforms-infrastructure-online-to-omnichannel.md]: fixed

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-e480443616/whitepapers/)

**Comment by nimbinatus** [comment]
LGTM, +1

**Pull request merged**